interface/code_systems/standard_tables_manage.php

```diff
@@ -83,6 +84,13 @@
             exit;
          }
     }
+    if ($version == "RF2International:English") {
```

Comment:
After here, change the version to International:English, since whether it is RF1 or RF2 should not matter in the entry that goes into databases (and is then displayed in the importer gui).